commands/addons.js

```diff
@@ -89,7 +89,7 @@ function* addonGetter(api, app) {
 }
 
 function displayAll(addons) {
-    addons = _.sortByAll(addons, 'app.name', 'plan.name', 'addon.name');
+    addons = _.sortBy(addons, 'app.name', 'plan.name', 'addon.name');
```

Comment:
it takes both, noted with the ellipses:
![screen shot 2016-03-31 at 4 27 48 pm](https://cloud.githubusercontent.com/assets/216188/14193803/e7444a1c-f75c-11e5-9912-d257f6c0a4a1.png)